Cubical/Algebra/Group/Abelianization/Properties.agda

```diff
@@ -111,29 +104,29 @@ module _ (G : Group ℓ) where
 
 
   -- Definition of the group structure on the abelianization. Here the generality of the comm relation is used.
-  _·Ab_ : Abelianization → Abelianization → Abelianization
+  _·Ab_ : Abelianization G → Abelianization G → Abelianization G
```

Comment:
Starting with this definition, things shouldn't be exported. Please put everything that follows in a (named) new module.